src/wolfio.c

```diff
@@ -694,7 +697,24 @@ int wolfIO_Recv(SOCKET_T sd, char *buf, int sz, int rdFlags)
 {
     int recvd;
 
+#ifdef FUSION_RTOS
```

Comment:
Please add a custom `wolfIO_Recv` and `wolfIO_Send` for `FUSION_RTOS` and do not modify the general one...

Reply:
Removed (customer ultimately used a custom I/O callback these changes were not necessary in the porting commit.